### DISCUSSION TITLE
Identification

### DISCUSSION THREAD

**Discussion by TerjeMaster** [rusoku/TouCAN-SocketCAN#9] [General]
Hi.

I guess if you have 10pcs TouCan-USB adapters connected to a linux computer at start, you never know which adapter gets can0, can1 etc...

Earlyer i tested peak-system's PCan-USB. Pcan has an ID-value which you can set to 0 - 255. It is stored in the firmware in the USB-Adapter.

Does TouCan-USB have this posbility?
If not,, Posability to add the feature?

Or is there any other way to secure that can0 allways get the correct TouCAN-USB adapter after every reboot?

Anyways. TouCAN-USB works great.

/TerjeMaster.

**Comment by hartkopp** [comment]
AFAIK there is a sysfs based interface for the PCAN devices which is on its way to be extended, see here:
 
https://lore.kernel.org/linux-can/PA4PR03MB67973D473C7CE600A6104EE8D6E39@PA4PR03MB6797.eurprd03.prod.outlook.com/
